tests/FSharp.Compiler.ComponentTests/EmittedIL/Attribute/Attribute.fs

```diff
@@ -0,0 +1,42 @@
+﻿namespace EmittedIL.RealInternalSignature
+
+open Xunit
+open FSharp.Test
+open FSharp.Test.Compiler
+
+module Attribute =
```

Comment:
On a similar note, could you include testing with aliases?
e.g. a non-generic type alias pointing to a generic instantiation, and then used as an attribute.
And vice versa.

```
type MyAttribute<^T>() =
    inherit Attribute()

type MyIntAttrAttribute = MyAttribute<int>

[<MyIntAttr>]
let x = 1

```

Reply:
this is a great idea, implementing now

Reply:
done